lms/djangoapps/course_wiki/plugins/__init__.py

```diff
@@ -0,0 +1,64 @@
+"""Module with the course app configuration for the Wiki."""
+from typing import Dict, Optional, TYPE_CHECKING
+
+from django.conf import settings
+from django.utils.translation import ugettext_noop as _
+from opaque_keys.edx.keys import CourseKey
+
+from openedx.core.djangoapps.course_apps.plugin import CourseApp
+
+if TYPE_CHECKING:
```

Comment:
Question: Why are you importing the user model conditionally? What is the reason for this "TYPE_CHECKING" condition?

Reply:
The user model is only used for type checking the code, and is not used by the code itself. 

Importing the User model here causes the platform to stop working since the model is imported before the Django machinery is ready.

Reply:
I think it's worth checking if it is because you have written the code in `__init__.py` file while for other apps the code is in `plugin.py`?

Reply:
There was an error raised which is why I made the change. This app already contains a plugins directory, so we can't also have a plugins.py file since that would be a namespace collision. 

This package is already use for wiki plugins so it must be imported somewhere.

Reply:
Oh is it already there? Let me take a look.

Reply:
I have verified that this is because you are importing this in the init file and the files is compiled during Django initialization time. If you move this to another file, you will not need this condition & you will be fine.

Reply:
Yup, I moved it here because you recommended that we have a uniform place for hosting this. 
There are a lot of reasons for doing the import this way, such as in case of expensive imports or circular import errors.  It doesn't really hurt performance, so I this it's a small price to pay for having a consistent place to import this from?

Reply:
Ok, Marking that an exception. I would like you to please add a comment describing the scenario in the file as well.
thanks.

Reply:
thanks.